vnext/Folly/Folly.vcxproj

```diff
@@ -283,7 +283,4 @@
     <Copy DestinationFiles="@(TemporaryFollyPatchFiles->'$(FollyDir)folly\%(RecursiveDir)%(Filename)%(Extension)')" SourceFiles="@(TemporaryFollyPatchFiles)" />
   </Target>
   -->
-  <Target Name="EnsureNodeModuleBuildImports" BeforeTargets="PrepareForBuild">
```

Comment:
We have the exact same target a few lines up. Thought I might as well remove this since I stumbled onto it.